xoinvader/ship.py

```diff
@@ -255,6 +266,22 @@ def __init__(self, pos, border, settings):
         self._fire = True
         self._wbay = Point(x=self._image.width // 2, y=1)
 
+        from xoinvader.animation import AnimationManager
+        self._animgr = AnimationManager()
+        self._animgr.add(
+            name="chaotic", bind=self, attr="direction",
```

Comment:
This is not related to mechanism itself. Animation binds to attribute in broad sense, not to `coordinate`. We have no sane component to move entities easily, this will be rewritted with new object system.
Direction is dirty thing now, but this is not delta.